openapi/ynab/client.bal

```diff
@@ -0,0 +1,401 @@
+import ballerina/http;
```

Comment:
license header missing

Reply:
Resolved & amended the changes. Thank you.